javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,112 @@
+// 'use strict';
+// Flight booking fullname function 
+
+function getFullname (firstname, surname, useFormalName = true, isfemale = true){
+    if (useFormalName === true && isfemale === false ){
+        return 'lord' + firstname + surname;
+    } else if (useFormalName === true && isfemale === true){
+        return 'Madam' + firstname + surname;
+    } else{
+        return firstname + surname;
+    }
+}
+const fullname1 = getFullname(' anas ', 'Barre')
+const fullname2 = getFullname( ' Anisa ', 'Barre')
+console.log(fullname1);
+console.log(fullname2);
+
+// Event application (hope this is right)
+
+function getEventWeekday (eventHeld) {
+    let day = new Date();
+    let today = day.getDay()
+    const days = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday']
+    const willBeHeld = days[(today + eventHeld) % 7]
+    return willBeHeld;
+}
+
+console.log('Event will be held on ' +  getEventWeekday(9));
+
+
+// Weather wear
+
+function weatherWear (temp) {
+    if (temp > 30){
+        console.log(`Shorts and a t-shirt`)
+    } else {
+        console.log(`its going to be a cold one consider wearing a jacket  and trousers`)
+    }
+}
+weatherWear(50)
```

Comment:
Good! And maybe you can add some more variables to consider more temperature conditions? Your function is suggesting jacket and trousers for a temperature of 25 degrees. 🥵🥵🥵